### DISCUSSION TITLE
doris be宕机

### DISCUSSION THREAD

**Discussion by supergenge** [apache/doris#16756] [A - General / Q&A]
*** Query id: 0-0 ***
*** Aborted at 1676304224 (unix time) try "date -d @1676304224" if you are using GNU date ***
*** Current BE git commitID: Unknown ***
*** SIGSEGV address not mapped to object (@0x7fcd9f4da000) received by PID 2704 (TID 0x7fcedf1c0700) from PID 18446744072087248896; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/doris/be/src/common/signal_handler.h:420
 1# os::Linux::chained_handler(int, siginfo*, void*) in /opt/software/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /opt/software/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /opt/software/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 4# 0x00007FCF30418400 in /lib64/libc.so.6
 5# phmap::priv::raw_hash_set<phmap::priv::FlatHashSetPolicy<doris::StringValue>, phmap::Hash<doris::StringValue>, phmap::EqualTo<doris::StringValue>, std::allocator<doris::StringValue> >::iterator phmap::priv::raw_hash_set<phmap::priv::FlatHashSetPolicy<doris::StringValue>, phmap::Hash<doris::StringValue>, phmap::EqualTo<doris::StringValue>, std::allocator<doris::StringValue> >::find<doris::StringValue>(doris::StringValue const&) at /var/local/thirdparty/installed/include/parallel_hashmap/phmap.h:1647
 6# doris::InListPredicateBase<(doris::PrimitiveType)23, (doris::PredicateType)7>::evaluate(doris::vectorized::IColumn const&, unsigned short*, unsigned short) const at /root/doris/be/src/olap/in_list_predicate.h:244
 7# doris::segment_v2::SegmentIterator::_evaluate_short_circuit_predicate(unsigned short*, unsigned short) at /root/doris/be/src/olap/rowset/segment_v2/segment_iterator.cpp:1073
 8# doris::segment_v2::SegmentIterator::next_batch(doris::vectorized::Block*) at /root/doris/be/src/olap/rowset/segment_v2/segment_iterator.cpp:1177
 9# doris::BetaRowsetReader::next_block(doris::vectorized::Block*) at /root/doris/be/src/olap/rowset/beta_rowset_reader.cpp:278
10# doris::vectorized::VCollectIterator::Level0Iterator::_refresh_current_row() at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:225
11# doris::vectorized::VCollectIterator::Level0Iterator::init(bool) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:207
12# doris::vectorized::VCollectIterator::build_heap(std::vector<std::shared_ptr<doris::RowsetReader>, std::allocator<std::shared_ptr<doris::RowsetReader> > >&) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:81
13# doris::vectorized::BlockReader::_init_collect_iter(doris::TabletReader::ReaderParams const&, std::vector<std::shared_ptr<doris::RowsetReader>, std::allocator<std::shared_ptr<doris::RowsetReader> > >*) at /root/doris/be/src/vec/olap/block_reader.cpp:65
14# doris::vectorized::BlockReader::init(doris::TabletReader::ReaderParams const&) at /root/doris/be/src/vec/olap/block_reader.cpp:130
15# doris::vectorized::NewOlapScanner::open(doris::RuntimeState*) at /root/doris/be/src/vec/exec/scan/new_olap_scanner.cpp:119
16# doris::vectorized::ScannerScheduler::_scanner_scan(doris::vectorized::ScannerScheduler*, doris::vectorized::ScannerContext*, doris::vectorized::VScanner*) at /root/doris/be/src/vec/exec/scan/scanner_scheduler.cpp:194
17# doris::ThreadPool::dispatch_thread() at /root/doris/be/src/util/threadpool.cpp:542
18# doris::Thread::supervise_thread(void*) at /root/doris/be/src/util/thread.cpp:455
19# start_thread in /lib64/libpthread.so.0
20# clone in /lib64/libc.so.6

**Comment by LemonLiTree** [comment]
which version

**Reply by supergenge** [reply]
doris-1.2.0-rc04

**Reply by supergenge** [reply]
Recently, a certain BE node has been frequently crashing in the past two days.

**Reply by supergenge** [reply]
*** Query id: 0-0 ***
*** Aborted at 1676390430 (unix time) try "date -d @1676390430" if you are using GNU date ***
*** Current BE git commitID: Unknown ***
*** SIGSEGV unkown detail explain (@0x0) received by PID 32684 (TID 0x7f3d643f2700) from PID 0; stack trace: ***
 0# doris::signal::(anonymous namespace)::FailureSignalHandler(int, siginfo_t*, void*) at /root/doris/be/src/common/signal_handler.h:420
 1# os::Linux::chained_handler(int, siginfo*, void*) in /opt/software/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 2# JVM_handle_linux_signal in /opt/software/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 3# signalHandler(int, siginfo*, void*) in /opt/software/jdk1.8.0_211/jre/lib/amd64/server/libjvm.so
 4# 0x00007F3DD59FA400 in /lib64/libc.so.6
 5# doris::BetaRowsetReader::support_return_data_by_ref() at /root/doris/be/src/olap/rowset/beta_rowset_reader.h:42
 6# doris::vectorized::VCollectIterator::Level0Iterator::init(bool) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:200
 7# doris::vectorized::VCollectIterator::build_heap(std::vector<std::shared_ptr<doris::RowsetReader>, std::allocator<std::shared_ptr<doris::RowsetReader> > >&) at /root/doris/be/src/vec/olap/vcollect_iterator.cpp:81
 8# doris::vectorized::BlockReader::_init_collect_iter(doris::TabletReader::ReaderParams const&, std::vector<std::shared_ptr<doris::RowsetReader>, std::allocator<std::shared_ptr<doris::RowsetReader> > >*) at /root/doris/be/src/vec/olap/block_reader.cpp:65
 9# doris::vectorized::BlockReader::init(doris::TabletReader::ReaderParams const&) at /root/doris/be/src/vec/olap/block_reader.cpp:130
10# doris::vectorized::NewOlapScanner::open(doris::RuntimeState*) at /root/doris/be/src/vec/exec/scan/new_olap_scanner.cpp:119
11# doris::vectorized::ScannerScheduler::_scanner_scan(doris::vectorized::ScannerScheduler*, doris::vectorized::ScannerContext*, doris::vectorized::VScanner*) at /root/doris/be/src/vec/exec/scan/scanner_scheduler.cpp:194
12# doris::ThreadPool::dispatch_thread() at /root/doris/be/src/util/threadpool.cpp:542
13# doris::Thread::supervise_thread(void*) at /root/doris/be/src/util/thread.cpp:455
14# start_thread in /lib64/libpthread.so.0
15# clone in /lib64/libc.so.6

**Reply by LemonLiTree** [reply]
等1.2.3 发了之后升级看下吧，可能是1.2.0的某个bug
